bundles/org.openhab.binding.nuvo/src/main/java/org/openhab/binding/nuvo/internal/handler/NuvoHandler.java

```diff
@@ -1114,6 +1119,20 @@ private void enableNuvonet(boolean showReady) {
                                     + topMenuItems.get(i).getText() + "\"");
                             Thread.sleep(SLEEP_BETWEEN_CMD_MS);
                         }
+
+                        // Build a State options selection that represents this source's custom menu
+                        List<StateOption> sourceMenuStateOptions = new ArrayList<>();
+                        topMenuItems.forEach(topItem -> {
+                            sourceMenuStateOptions.add(new StateOption(topItem.getText(), topItem.getText()));
+                            topItem.getItems().forEach(subItem -> {
```

Comment:
If they is only one statement, you can avoid using `{....;}`

Reply:
fixed